requirements/requirements-test.txt

```diff
@@ -1,4 +1,4 @@
-tox>=2.1.1
-pytest>=2.8.2
-pytest-django>=2.9.0
-factory_boy>=2.6.0
+tox==3.3.0
+pytest==3.8.0
+pytest-django==3.4.2
+factory_boy==2.11.1
```

Comment:
I think we need to add flake8 to the requirements.

Reply:
okie dokie